<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDI Application (PR11 - SQL)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnCreateSong">
      <property name="text">
       <string>Створити Пісню</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateBook">
      <property name="text">
       <string>Створити Книгу</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnShowSongs">
      <property name="text">
       <string>Показати/Приховати Пісні</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnShowBooks">
      <property name="text">
       <string>Показати/Приховати Книги</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Вихід</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
